[outdated - OK to close?] Added items, descriptions, and author

Myndex opened the pull request:
Added some items such as for CVD, and added some description for some, added myself as author on those items that I am focused on already.  

Made the second column for descriptions 2-wide, and put links into \<a\> refs to prevent the word wrap problem that made a column too wide.

Myndex committed:
Added items, descriptions, and author

Added some items and added some description for some. made the second column for descriptions 2-wide, and put links into a refs to prevent the word wrap problem.

shawna-slh commented:
Hi Andy. Unfortunately we could not merge this because the code was incorrect and it messed up the table.

Also, in the column "Applicability to technology, user agents, or authors" you had put your name several places.

I think I got your name added as an editor to most of the things you wanted.

+1 to changing the uris to ```<a>``` &mdash; I shortened them even further.

I did some more to shorten the required width for columns. And help avoid horizontal scrolling for more users.

I also worked on making the short description even shorter. And moved the details to a new GitHub Discussion for a couple of them. I think Jonathan or Chris will send more guidance on that.

I think it might be easiest to do further editing of this live together. I will check with Chris and Jonathan on approach for that.

Myndex commented:
Hi @shawna-slh 

> Hi Andy. Unfortunately we could not merge this because the code was incorrect and it messed up the table.

It displayed correctly on the multiple browsers I tested it on, nothing was messed up that I saw, can you give me a specific?

> Also, in the column "Applicability to technology, user agents, or authors" you had put your name several places.

I see this as a mistake now, it was confusing at the time, as it "looked like" this meant authors as in those working on the _supplement_, not the content creators that the supplemental guidance is _directed at._ This would have been an easy and quick fix for me had it been brought to my attention.

SIDE NOTE: If I've never mentioned this, I utterly loath that web content creators are referred to here as "authors" as it only creates ambiguity IMO.

> I think I got your name added as an editor to most of the things you wanted.

Nothing I added is here in the pull request, not names nor the additional items I added.

> +1 to changing the uris to `<a>` — I shortened them even further.

Yes, I saw that, appears to be the only change carried over.

> I did some more to shorten the required width for columns. And help avoid horizontal scrolling for more users.

One of what I had done is set the description column to two units wide, to facilitate reading. The next step was going to be to move over anything from other columns that were "crushed".


> I think it might be easiest to do further editing of this live together. I will check with Chris and Jonathan on approach for that.

From the meeting, I did not take this to be any form of a normative document, more as an open place to drop in short idea summaries. 

### SECOND SIDE NOTE: _If you'd like to know the best way to really annoy an author, it is to delete their work without comment nor giving them at least one chance to correct it to make it conform to expectations._

Not that this rises even to "work" — but If this is a collaboratory group and process, then we need to discuss methods of iteration and editing of content. A good functional example is that over at Mozilla, where the editorial process is more akin to the traditional one of a back-and-forth, with an objective eye pointing out changes that are needed, but not changing without the opportunity of input from the originator.

A more extreme example is that here in Hollywood: producers are not allowed to even _see_ the film they are producing for ten weeks after shooting is complete. This is to give the director and editor time to work with the material and present the work before the producers are even involved, at which time the typical workflow is the producer giving "_notes_ of things that need to be addressed".

Collaborative environments require some boundaries, and this one is key.

If you felt that what I added was too lengthy or placed in the wrong "slots" then I would have appreciated a quick note to that effect on the pull request, to which I could make adjustments to conform. What is here now is not conformed, all items added were _deleted_. This is apparently an issue here at the W3 that I and other collaborators have seen and been struck by, and I am voicing my objection as it is not appropriate for a collaborative environment.

Thank you,

Andy

shawna-slh commented:
Sorry for the long delay in replying. The last few weeks (and likely upcoming days or weeks) I have had little availability due to family medical emergencies and on-going complications.

> Hi Andy. Unfortunately we could not merge this because the code was incorrect and it messed up the table.
>> It displayed correctly on the multiple browsers I tested it on, nothing was messed up that I saw, can you give me a specific?

This is what I saw -- extra cells in third row that threw off the data and added additional columns:
![image](https://user-images.githubusercontent.com/6991814/138482925-72ce454b-0e7a-4518-871d-01ebc1daa394.png)

So I just now ran validator:
* Error: Stray end tag a. From line 71, column 87; to line 71, column 90
* Error: End tag td seen, but there were open elements. From line 71, column 91; to line 71, column 95
* Error: Unclosed element ul. From line 68, column 24; to line 68, column 27
* Error: Stray end tag ul. From line 71, column 96; to line 72, column 7
* Error: Stray end tag ul. From line 71, column 96; to line 72, column 7
* Error: Stray end tag td. From line 72, column 8; to line 72, column 12
* Error: Table column 3 established by element th has no cells beginning in it. From line 38, column 48; to line 39, column 33

<hr>

> Use of color (hue/sat) - Color as in hue/chroma/saturation are not used as the sole means of presenting information. This relates to the user needs of those with Color Vision Deficiency, as well as COGA needs relating to cultural differences in color meanings.

Not clear how this is different from 1.4.1 https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html ?

> Alternative Contrast Guideline

APCA now included in https://github.com/w3c/low-vision-a11y-tf/discussions/129

<hr>

> I think I got your name added as an editor to most of the things you wanted.

You are listed as Lead Editor for:
* Text at edge of viewport
* Disabled Components have sufficient contrast
* Enhanced Contrast
* Minimum background brightness
* Support Local adaptation
* Scaling for resize of text for large type elements

<hr>

I apologize that you felt the Pull Request was closed inappropriately. I will not be so diligent at housekeeping.

Now it is re-opened.
